src/tmx_xml.c

```diff
@@ -343,6 +344,16 @@ static int parse_layer(xmlTextReaderPtr reader, tmx_layer **layer_headadr, int m
 		tmx_free_func(value);
 	}
 
+	if ((value = (char*)xmlTextReaderGetAttribute(reader, (xmlChar*)"x"))) { /* x_offset */
+		res->x_offset = (int)strtod(value, NULL);

```

Comment:
Actually... I don't remember why exactly I used strtod over atoi. I might have had a special reason for this, but I don't remember now. Might be something as simple as "first result I got from Google search". I guess replacing it by atoi should be fine.